experimental/dds/tree2/src/simple-tree/proxies.ts

```diff
@@ -874,13 +874,19 @@ interface ExtractedFactoryContent<T extends InsertableTypedNode<TreeNodeSchema>>
 export function extractFactoryContent<
```

Comment:
Unrelated: Line 866 may have a typo, should it be `(x);` at the end?

Reply:
Yes, that is an error. I'm including the fix as part of https://github.com/microsoft/FluidFramework/pull/18630.